src/App.js

```diff
@@ -168,6 +155,7 @@ function App(props) {
             I18n.setLanguage(user[0].defaultLanguage);
           }
           const initialData = await Promise.all([
```

Comment:
Separated the process of fetching the initial sprint data from the process of connecting to the websocket. Might require a little refactoring in Sprints.js or SprintCreation.js but I can open a ticket for that if so.

Reply:
"Might require a **little** refactoring in Springs.js" = famous last words 😆